src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -34,7 +34,7 @@
         id="btn-add-person"
         outlined
         color="primary"
-        :disabled="showOrgPersonForm"
+        :disabled="renderOrgPersonForm"
```

Comment:
I renamed this because we don't show/hide (using `v-show`) the form, but we render/unrender (using `v-if`).